test/function.cc

```diff
@@ -6,6 +6,18 @@ namespace {
 
 int testData = 1;
 
+Boolean EmptyConstructor(const CallbackInfo& info) {
```

Comment:
Mybe it's possible to handle this case with one function instead of the two `EmptyConstructor` and `NonEmptyConstructor` something like this:
```cpp
Boolean EmptyConstructor(const CallbackInfo& info) {
  auto env = info.Env();
  bool is_empty = info[0].As<Boolean>();
  Function function = is_empty ? Function(): Function(env, Object::New(env));
  return Boolean::New(env, function.IsEmpty());
}
```
and from the JavaScript side:

```js
assert.strictEqual(binding.emptyConstructor(true), true);
assert.strictEqual(binding.emptyConstructor(false), false);
```

Reply:
Sure: https://github.com/nodejs/node-addon-api/pull/928/commits/da75b85dee3deec54a535f1264ca93156e65387c